Assets/Scripts/GameController.cs

```diff
@@ -16,6 +16,7 @@ public class GameController : MonoBehaviour
     public static GameController instance;
 
     public GameObject satellite;
+    public GameObject player;
```

Comment:
I'd name this class playerPrefab because it will be used to instantiated instead. Player is too generic and it is unknown whether you're talking about the player in game already a new a one